frontend/src/components/single-contract/DisputeResolverForm.tsx

```diff
@@ -0,0 +1,195 @@
+import { useJob } from 'components/smart-contracts/hooks/useJob';
+import { ResolveEngineerModal } from 'components/smart-contracts/modals/ResolveEngineerModal';
+import { ResolveSplitModal } from 'components/smart-contracts/modals/ResolveSplitModal';
+import { ResolveSupplierModal } from 'components/smart-contracts/modals/ResolveSupplierModal';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+export const DisputeResolverForm = () => {
```

Comment:
Can we break this file up into more components? Like on per row, they don't need to be reusable. It's just hard to tell whats happening in here.

Reply:
Will do